### DISCUSSION TITLE
提一个 Vue props 类型标注的问题

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#2808] [提问 / Q&A] [answered]
在 `Bilibili-Evolved\src\components\settings-panel\mixins.ts` 中的第 7 行，`type: Object as () => ComponentMetadata` 这种类型标注是否是错误的？我在 Vue 的官方文档中没有看到过这种用法，感觉似乎不太对劲。

**Comment by the1812** [comment, marked as answer]
忘了在哪看来的了, 反正写对了也没有类型提示的, 无所谓了
